api_tests/nodes/views/test_node_list.py

```diff
@@ -96,6 +98,25 @@ def test_omit_retracted_registration(self):
         res = self.app.get(self.url, auth=self.user.auth)
         assert_equal(len(res.json['data']), 2)
 
+    def test_node_list_has_root(self):
+        res = self.app.get(self.url, auth=self.user.auth)
+        projects_with_root = 0
+        for project in res.json['data']:
+            if project['relationships'].get('root', None):
+                projects_with_root += 1
+        assert_equal(projects_with_root, 2)
```

Comment:
Shouldn't all projects have a root? If so, then it might be more useful to assert that

```python
assert_true(all([each['relationships'].get('root') is not None for each in res.json['data']]))
```

That would conveniently get rid of the need to rely on a count (`projects_with_root`), which is brittle.